modules/gapi/src/streaming/onevpl/source_priv.cpp

```diff
@@ -261,7 +266,7 @@ GSource::Priv::~Priv() {
     GAPI_LOG_INFO(nullptr, "Unload MFX implementation description: " << mfx_impl_description);
     MFXDispReleaseImplDescription(mfx_handle, mfx_impl_description);
     GAPI_LOG_INFO(nullptr, "Unload MFX handle: " << mfx_handle);
-    MFXUnload(mfx_handle);
+    //MFXUnload(mfx_handle);
```

Comment:
M B it will be uncommented in the future.

Reply:
truth